send-emails-from-a-service/main.bal

```diff
@@ -0,0 +1,164 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/email;
+import ballerina/jballerina.java;
+import ballerina/uuid;
+
+type Patient record {|
+    string name;
+    string dob;
+    string ssn;
+    string address;
+    string phone;
+    string email;
+|};
+
+type Doctor record {|
+    string name;
+    string hospital;
+    string category;
+    string availability;
+    decimal fee;
+|};
+
+type Appointment record {|
+    int appointmentNumber;
+    Doctor doctor;
+    Patient patient;
+    boolean confirmed;
+    string hospital;
+    string appointmentDate;
+|};
```

Comment:
@MaryamZi Need your input as well. 

We better keep names consistent. In the "simple message..." tutorial we used the same name (`Appointment`) I have used `ReservationResponse` in my "content-based message ...". What name we should use and move forward with?